docs/ref/contrib/gis/serializers.txt

```diff
@@ -38,7 +38,8 @@ serializers.
 Example::
 
     from django.core.serializers import serialize
-    from my_app.models import City
+
+    from .models import City
```

Comment:
I would leave this alone since it's not obvious this example lives in a certain file. It looks more like a shell example.

Reply:
ok